pallets/proofs-dealer/src/lib.rs

```diff
@@ -484,10 +484,14 @@ pub mod pallet {
         /// This is the generic version of [`MutationsAppliedForProvider`](Event::MutationsAppliedForProvider)
         /// when [`generic_apply_delta`](ProofsDealerInterface::generic_apply_delta) is used
         /// and the root is not necessarily linked to a specific Provider.
+        ///
+        /// Additional information for context on where the mutations were applied can be provided
+        /// by using the `event_info` field.
```

Comment:
I would point to the common encode and decode functions if you agree to add them

Reply:
Yes that makes sense.